doc/mgr/administrator.rst

```diff
@@ -106,17 +106,17 @@ Here is an example of enabling the :term:`Dashboard` module:
 	{
 		"dashboard": "http://myserver.com:7789/",
 		"restful": "https://myserver.com:8789/"
-	}
+	``
```

Comment:
nit: looks like this change crept in somehow?

Reply:
thanks. reverted.